client/landing/gutenboarding/index.tsx

```diff
@@ -129,11 +122,38 @@ window.AppBoot = async () => {
 					</Route>
 				</Switch>
 			</BrowserRouter>
-		</I18nProvider>,
+		</LocaleContext>,
 		document.getElementById( 'wpcom' )
 	);
 };
 
+function WindowLocaleEffectManager() {
```

Comment:
Similarly to https://github.com/Automattic/wp-calypso/pull/43132/files#r444074316 this function could do with more re-usable friendly location.

Reply:
Thanks, I pulled that out to a separate component 👍